DSharpPlus/Entities/Channel/Overwrite/DiscordOverwriteBuilder.cs

```diff
@@ -72,7 +72,7 @@ public DiscordOverwriteBuilder(DiscordRole role)
         /// <summary>
         /// Creates a new Discord permission overwrite builder. This class can be used to construct permission overwrites for guild channels, used when creating channels.
         /// </summary>
-        [Obsolete("Will be removed in 5.0. Use specialized constructors instead", false)]
+        [Obsolete("The parameterless constructor will be removed in 4.4. Use the specialized constructors instead", false)]
```

Comment:
This should likely say *parameterized* constructors, given the confusion that's spurred from the wording.

Reply:
good idea, will update